app/products/playbooks/actions/websocket/events.test.ts

```diff
@@ -0,0 +1,151 @@
+        expect(handlePlaybookPluginEnabled).not.toHaveBeenCalled();
+        expect(handlePlaybookRunCreated).not.toHaveBeenCalled();
+        expect(handlePlaybookRunUpdated).not.toHaveBeenCalled();
+        expect(handlePlaybookRunUpdatedIncremental).not.toHaveBeenCalled();
+    });
+
+    it('should handle WEBSOCKET_PLAYBOOK_RUN_CREATED event', async () => {
+        const msg = TestHelper.fakeWebsocketMessage({
+            event: WEBSOCKET_EVENTS.WEBSOCKET_PLAYBOOK_RUN_CREATED,
+            data: {
+                payload: JSON.stringify({
+                    playbook_run: {id: 'run1'},
+                }),
+            },
+        });
+
+        await handlePlaybookEvents(serverUrl, msg);
+
+        expect(handlePlaybookRunCreated).toHaveBeenCalledWith(serverUrl, msg);
+        expect(handlePlaybookRunCreated).toHaveBeenCalledTimes(1);
+        expect(handlePlaybookRunUpdated).not.toHaveBeenCalled();
+        expect(handlePlaybookRunUpdatedIncremental).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginEnabled).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginDisabled).not.toHaveBeenCalled();
+    });
+
+    it('should handle WEBSOCKET_PLAYBOOK_RUN_UPDATED event', async () => {
+        const msg = TestHelper.fakeWebsocketMessage({
+            event: WEBSOCKET_EVENTS.WEBSOCKET_PLAYBOOK_RUN_UPDATED,
+            data: {
+                payload: JSON.stringify({
+                    id: 'run1',
+                }),
+            },
+        });
+
+        await handlePlaybookEvents(serverUrl, msg);
+
+        expect(handlePlaybookRunUpdated).toHaveBeenCalledWith(serverUrl, msg);
+        expect(handlePlaybookRunUpdated).toHaveBeenCalledTimes(1);
+        expect(handlePlaybookRunCreated).not.toHaveBeenCalled();
+        expect(handlePlaybookRunUpdatedIncremental).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginEnabled).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginDisabled).not.toHaveBeenCalled();
+    });
+
+    it('should handle WEBSOCKET_PLAYBOOK_RUN_UPDATED_INCREMENTAL event', async () => {
+        const msg = TestHelper.fakeWebsocketMessage({
+            event: WEBSOCKET_EVENTS.WEBSOCKET_PLAYBOOK_RUN_UPDATED_INCREMENTAL,
+            data: {
+                payload: JSON.stringify({
+                    id: 'run1',
+                    changed_fields: {},
+                }),
+            },
+        });
+
+        await handlePlaybookEvents(serverUrl, msg);
+
+        expect(handlePlaybookRunUpdatedIncremental).toHaveBeenCalledWith(serverUrl, msg);
+        expect(handlePlaybookRunUpdatedIncremental).toHaveBeenCalledTimes(1);
+        expect(handlePlaybookRunCreated).not.toHaveBeenCalled();
+        expect(handlePlaybookRunUpdated).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginEnabled).not.toHaveBeenCalled();
+        expect(handlePlaybookPluginDisabled).not.toHaveBeenCalled();
+    });
+
+    it('should handle unknown event by doing nothing', async () => {
```

Comment:
when processing an unknown event there should be a log, but I'm not sure if that's easy to test (maybe mocking the log system and checking it was called?)

0/5 on testing this